bootstrap.js

```diff
@@ -7,54 +7,61 @@ const index = fs.readFileSync('index.html', 'utf8');
 const ResponseBuilder = require('./app/ResponseBuilder');
 
 module.exports = app => {
-    app.get('/*', (req, res) => {
+    app.all('/*', (req, res) => {
         const responseBuilder = new ResponseBuilder(res);
-        
+
         const requestedUrl = req.url.slice(1);
         const corsBaseUrl = '//' + req.get('host');
-        
-        console.info(req.protocol + '://' + req.get('host') + req.url);
-        
+
+        console.info(req.method + ' ' + req.protocol + '://' + req.get('host') + req.url);
+
         if(requestedUrl == ''){
             res.send(index);
             return;
         }
 
+        let headers = Object.assign({}, req.headers);
+
+        // This prevents "Error [ERR_TLS_CERT_ALTNAME_INVALID]: Hostname/IP does not match certificate's altnames: Host: localhost"
+        delete headers.host;
```

Comment:
Could we ensure this functionality is tested (e.g that the `headers.host` is correctly deleted and the `headers['accept-encoding']` also).

Reply:
This would make a lot of sense.